api/src/gameState/gameState.controller.ts

```diff
@@ -54,7 +90,7 @@ export class GameStateController {
       const game = await this.gameStateService.getGame({ id });
       return this.gameStateService.deserializerGame(game);
     } catch (error) {
-      throw new NotFoundException("That game does not exists");
+      throw new NotFoundException('That game does not exists');
```

Comment:
That games does not exist. Watch out of your English.

Reply:
I think we say "Watch out **for** your english" 😛  so... Watch out for your english @mchaffotte 😃